lib/ckb/always_success_wallet.rb

```diff
@@ -16,14 +16,14 @@ def send_capacity(target_address, capacity)
       outputs = [
         {
           capacity: capacity,
-          data: [],
+          data: "",
```

Comment:
What made us change data from `[uint8]` to `String`?

Reply:
Data be packed to binary string, such as `[i.amounts - amount].pack("Q<")`,
also see `Ckb::Utils.calculate_cell_min_capacity`  `output[:data].bytesize`

Reply:
I think right now CKB's RPC still only accepts array of integers as `Vec<u8>` correct? Or have we changed that?

Reply:
Ok, I am confused with two format, so we use binary string for sha, array byte for RPC request ?
I will revert this, leave it for the future.